helm/charts/determined/values.schema.json

```diff
@@ -0,0 +1,101 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "properties": {
+      "region": {
+        "type": "string",
+        "enum": [
+          "ORD1",
+          "EWR1",
+          "LAS1",
+          "LGA1"
+        ],
+        "form": true,
+        "title": "Region",
+        "customComponent": {
+          "id": "region_selector",
+          "component": "RegionSelector"
+        }
+      },
+      "resources": {
+        "type": "object",
+        "title": "Resources",
+        "form": true,
+        "properties": {
+          "cpu": {
+            "type": "string",
+            "form":"true",
+            "title": "cpu",
+            "description": "Amount of CPU to provide",
+            "render":"slider",
+            "sliderMin": 8000,
+            "sliderMax": 32000,
+            "sliderUnit": "m",
+            "default": "8000m"
+          },
+          "memory": {
+              "type": "string",
+              "form":"true",
+              "title": "memory",
+              "description": "Amount of Memory to provide",
+              "render":"slider",
+              "sliderMin": 8000,
+              "sliderMax": 32000,
+              "sliderUnit": "Mi",
+              "default": "32000Mi"
+          },
+          "gpu_type": {
+            "type": "string",
+            "enum": [
+              "RTX_A4000",
+              "RTX_A5000",
+              "RTX_A6000",
+              "A100_NVLINK",
+              "A100_PCIE_40GB",
+              "A100_PCIE_80GB",
+              "A40",
+              "Tesla_V100_NVLINK",
+              "Quadro_RTX_5000",
+              "Quadro_RTX_4000"
+            ],
+            "form": true,
+            "title": "GPU Type",
+            "default": "RTX_A5000",
+            "description": "Please select the default GPU type for your workloads."
+          }
+        }
+      },
+      "s3": {
+        "type": "object",
+        "form": "true",
+        "title": "S3 Configuration",
+        "properties": {
+          "bucket": {
+            "type": "string",
+            "form": "true", 
+            "title": "Bucket Name",
+            "description": "The name of the S3 bucket to store Model Checkpoints."
+          },
+          "accessKey": {
+            "type": "string",
+            "form": "true", 
+            "title": "Access Key",
+            "description": "Access Key for S3"
+          },
+          "secretKey": {
+            "type": "string",
+            "form": "true", 
+            "title": "Secret Key",
+            "description": "Secret Key for S3"
+          },
+          "endpointUrl": {
+            "type": "string",
+            "form": "true", 
+            "title": "Endpoint URL",
+            "description": "Endpoint URL for S3"
```

Comment:
What is this? Is both bucket name and this required?

Reply:
unless we hardcode the value to `s3.ord1.coreweave.com` etc. it needs to be specified somewhere.

Reply:
Nope, removing it.

Reply:
@brandonrjacobs yeah I am trying to understand if the full domain is part of bucketname or not. If not, we probably want to leave it. and add a description: `Ie. s3.ord1.coreweave.com`